tutorials/roofit/rf614_binned_fit_problems.C

```diff
@@ -116,8 +116,8 @@ void rf614_binned_fit_problems()
    // trick by sampling the integral in the same way doesn't cancel out the
    // bias completely. The average bias is canceled, but there are per-bin
    // biases that remain. Still, this method has some value: it is cheaper than
-   // rigurously correcting the bias by integrating the pdf in each bin. So if
-   // you know your per-bin bias variations are small or performance is an
+   // rigorously correcting the bias by integrating the pdf in each bin. So if
+   // you know your per-bin bias, variations are small or performance is an
```

Comment:
Actually I meant exactly the variance of the "per-bin bias" here, so the comma doesn't belong here.

Maybe it's clearer to say "if you know that the bias in not that different for each bin" or something else that avoids the term "variance" or "variations" that was probably confusing here